3817_Sprint/3-Sheik/demo/src/main/java/com/qualogy/demo/App.java

```diff
@@ -0,0 +1,72 @@
+package com.qualogy.demo;
+
+import bulbasaur.Bulbasaur;
+import charmander.Charmander;
+import ditto.Ditto;
+import pikachu.Pikachu;
+import ispokemon.isPokemon;
+import squirtle.Squirtle;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+public class App {
+
+    String SPACE = " ";
+    Random RANDOM = new Random();
+    Scanner SCANNER = new Scanner(System.in);
+
+    List<isPokemon> pokemonList = new ArrayList<>();
+    isPokemon pokemon;
+
+    public static void main(String[] args) {
+        App app = new App();
+    }
+
+    public App() {
+        startGame();
+        addPokemon();
+        pickPokemon();
+        displayOptions();
+        playerTurn();
+    }
+
+    public void startGame() {
+        System.out.println("Who is that pokemon?!");
+    }
+
+    public void addPokemon() {
```

Comment:
addPokemons()? plural version fits better

Reply:
The pural of pokemon is pokemon..